### PR TITLE
Test simple http request to exposed minio in rafter integration tests

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Test simple http request to exposed minio in rafter integration tests: try fetch file

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/6907

### CODE REVIEW

File: tests/rafter/pkg/file-client/file-client.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
+package fileclient
```
Comment (michal-hudy) [inline tests/rafter/pkg/file-client/file-client.go]: Don't use `-` in package/direcotry names

File: tests/rafter/pkg/file-client/file-client.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+type Config struct {
+	Address   string `envconfig:"default=storage.kyma.local"`
+	Secure    bool   `envconfig:"default=true"`
```
Comment (michal-hudy) [inline tests/rafter/pkg/file-client/file-client.go]: Needed? Use address and protocol from Asset status.

File: tests/rafter/pkg/file-client/file-client.go [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+	if len(splitBaseURL) < 3 {
+		return ""
+	}
+
+	bucketName := splitBaseURL[len(splitBaseURL)-3]
+	assetName := splitBaseURL[len(splitBaseURL)-2]
+	fileName := splitBaseURL[len(splitBaseURL)-1]
+
+	return fmt.Sprintf("%s/%s/%s/%s", f.endpoint, bucketName, assetName, fileName)
+}
```
Comment (michal-hudy) [inline tests/rafter/pkg/file-client/file-client.go]: Is this needed? You have a proper address to MinIO in Asset status.

File: tests/rafter/pkg/file-client/file-client.go [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+func (f *FileClient) fetch(url string) ([]byte, error) {
+	if url == "" {
+		return nil, nil
+	}
+
+	resp, err := f.client.Get(url)
```
Comment (michal-hudy) [inline tests/rafter/pkg/file-client/file-client.go]: To tests CORS, you should set `Origin` header.

Comment (magicmatatjahu) [reply tests/rafter/pkg/file-client/file-client.go]: Added

File: tests/rafter/pkg/file-client/file-client.go (hunk 3 above)
Comment (magicmatatjahu) [reply tests/rafter/pkg/file-client/file-client.go]: Removed :)

File: tests/rafter/pkg/file-client/file-client.go (hunk 2 above)
Comment (magicmatatjahu) [reply tests/rafter/pkg/file-client/file-client.go]: Done :)

**Pull request merged**